content/components/page-header.mdx

```diff
@@ -0,0 +1,330 @@
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Parent link
+    </Text>
+    <Text as="p" mt="0">
+        The default way to let users navigate up in the hierarchy on narrow viewports.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222194471-dfc56e9a-a2ab-48a0-8a04-41ebb5ec4adf.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Context bar actions
+    </Text>
+    <Text as="p" mt="1">
+        Actions that appear in the Context bar specifically.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222194470-da8126b8-72c5-4078-968e-ebc5c2bf3140.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Context bar area
+    </Text>
+    <Text as="p" mt="0">
+        The Context bar is a generic slot for any component above the title region. 
+        Use it for custom breadcrumbs and other navigation elements instead of Parent link.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222194469-85131849-4a7a-41e6-abca-040d042110df.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+### Context bar responsive strategy
+
+On narrow viewports, the Parent link touch target expands beyond its element boundaries.
+
+<img 
+    src="https://user-images.githubusercontent.com/586552/222216317-c8478ab0-6541-49c6-9140-bc884e9a8cd4.png"
+    role="presentation"
+    width="877"
+    alt="Page header context bar in a narrow viewport"
+/>
+
+## Title bar
+
+The Title bar can include a static page title, actions, and a parent link that by default is only visible on narrow viewports.
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Default title variant
+    </Text>
+    <Text as="p" mt="0">
+        The default way to let users navigate up in the hierarchy on narrow viewports.
```

Comment:
So this title is also a link? It's not really clear in the example image. It just looks like this is the "default" text size/style for the title.

Reply:
I'm not sure what you mean here ^